client/src/components/PayBill.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useRef, useEffect, useState } from 'react';
+import $ from 'jquery';
+import ThankYou from './ThankYou.jsx';
+
+// const payeeEmail = 'sb-dmus17722352@personal.example.com';
+// const hostName = 'Milo';
+// const billInfo = [
+//   {
+//     name: 'Milo',
+//     order: [
+//       { reference_id: 1, description: 'Milo"s order', amount: { currency_code: 'USD', value: 135.3 } }]
+//   },
+//   {
+//     name: 'Sara',
+//     order: [
+//       { reference_id: 4, description: 'Sara"s order', amount: { currency_code: 'USD', value: 25.3 } }]
+//   },
+//   {
+//     name: 'Tom',
+//     order: [
+//       { reference_id: 6, description: 'Tom"s order', amount: { currency_code: 'USD', value: 52.3 } }]
+//   },
+//   {
+//     name: 'Jorge',
+//     order: [
+//       { reference_id: 5, description: 'Jorge"s order', amount: { currency_code: 'USD', value: 15.3 } }]
+//   }
+// ];
+
+const billInfoGenerator = (paymentsOwed) => {
+  let billInfo = [];
+  let i = 0;
+  for (let key in paymentsOwed) {
+    let result = {};
+    let order = {
+      reference_id: i + Date.now(),
+      description: key + '"s order',
+      amount: { currency_code: 'USD', value: Number(paymentsOwed[key]) }
+    };
+    result.name = key;
+    result.order = [];
+    result.order.push(order);
+    billInfo.push(result);
+    i++;
+  }
+  return billInfo;
+}
+
+const BillCard = (props) => (
+  <div className='card bill-card' >
+    <input type="checkbox" className="btn-check" id={'btncheck' + props.index} autoComplete="off" onClick={props.onClick} />
+    <label className="btn btn-outline-danger" htmlFor={'btncheck' + props.index}>{props.info.name + ' $' + props.info.bill}</label>
+  </div>
+);
+
+const PayBill = (props) => {
```

Comment:
This all looks great! There's still a few console.logs - no rush but would be good to clean those up later.